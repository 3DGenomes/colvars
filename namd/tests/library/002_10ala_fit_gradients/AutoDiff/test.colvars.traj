# step         eigenvector_Ca        fs_eigenvector_Ca     fa_eigenvector_Ca    
           0    1.00000000000000e+00  0.00000000000000e+00 -1.00000000000000e+01
