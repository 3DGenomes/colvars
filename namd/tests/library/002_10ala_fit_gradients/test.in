colvarsTrajFrequency    1

indexFile ../Common/da.ndx


colvar {
    name eigenvector_Ca

    outputTotalForce on
    outputAppliedForce on
    width 1.0 

    eigenvector {
        #    debugGradients
        atoms {
            enableFitGradients yes
            indexGroup Protein_C-alpha
        }
        refpositionsfile ../Common/pi-ideal.xyz
        #    vector { (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) }
        vectorFile ../Common/da.xyz
        differenceVector yes
    }
} 


harmonic {
    colvars        eigenvector_Ca
    forceConstant  10.0  
    centers        0.0
}
