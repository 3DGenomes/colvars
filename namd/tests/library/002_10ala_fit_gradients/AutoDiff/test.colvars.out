colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  http://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is available.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "../Common/da.ndx"
colvars: The following index groups were read from the index file "../Common/da.ndx":
colvars:   Protein (104 atoms).
colvars:   Protein_noH (51 atoms).
colvars:   Protein_Backbone (40 atoms).
colvars:   Protein_C-alpha (10 atoms).
colvars:   RMSD_atoms (10 atoms).
colvars:   Protein_C-alpha_1_2 (2 atoms).
colvars:   Protein_C-alpha_9_10 (2 atoms).
colvars:   Protein_C-alpha_1 (1 atoms).
colvars:   group1 (4 atoms).
colvars:   Protein_C-alpha_2 (1 atoms).
colvars:   group2 (4 atoms).
colvars:   Protein_C-alpha_3 (1 atoms).
colvars:   group3 (4 atoms).
colvars:   Protein_C-alpha_4 (1 atoms).
colvars:   group4 (4 atoms).
colvars:   Protein_C-alpha_5 (1 atoms).
colvars:   group5 (4 atoms).
colvars:   Protein_C-alpha_6 (1 atoms).
colvars:   group6 (4 atoms).
colvars:   Protein_C-alpha_7 (1 atoms).
colvars:   group7 (4 atoms).
colvars:   Protein_C-alpha_8 (1 atoms).
colvars:   group8 (4 atoms).
colvars:   Protein_C-alpha_9 (1 atoms).
colvars:   group9 (4 atoms).
colvars:   Protein_C-alpha_10 (1 atoms).
colvars:   group10 (4 atoms).
colvars:   heavy_atoms (51 atoms).
colvars: # smp = on [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "eigenvector_Ca"
colvars:   Initializing a new "eigenvector" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "Protein_C-alpha"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/pi-ideal.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating the vector projection: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     # vector =  [default]
colvars:     # vectorFile = "../Common/da.xyz"
colvars:     # vectorCol = "" [default]
colvars:     Geometric center of the provided vector: ( 0.9999 , 2 , 3 )
colvars:     # differenceVector = on
colvars:     "differenceVector" is on: subtracting the reference positions from the provided vector: v = v - x0.
colvars:     "differenceVector" is on: normalizing the vector.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0 [default]
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # upperWallConstant = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { eigenvector_Ca }
colvars:   # outputEnergy = off [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # forceConstant = 10
colvars:   # targetForceConstant = 0 [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group eigenvector_Ca:0/0. 10 atoms: total mass = 120.11.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group eigenvector_Ca:0/0. 10 atoms: total mass = 120.11.
colvars: The final output state file will be "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
